packages/material-ui/src/Button/Button.js

```diff
@@ -131,13 +130,19 @@ export const styles = theme => ({
     padding: 0,
     minWidth: 0,
     width: 56,
-    fontSize: 24,
     height: 56,
     boxShadow: theme.shadows[6],
     '&:active': {
       boxShadow: theme.shadows[12],
     },
   },
+  extended: {
```

Comment:
Something is off:
- property name: `extended-fab`
- class name: `extended`

Maybe using `extendedFab`? for the variant name?